packages/runtime/src/ctx.ts

```diff
@@ -163,36 +163,52 @@ export class Ctx implements Context {
   }
 
   /**
-   * Add asset using asa.yaml file
+   * Deploy asset using asa.yaml file
    * @param name asset name
    * @param fromAccountAddr account address
    * @param flags asa deployment flags
    */
-  addAsset (
+  deployASA (
     name: string,
     fromAccountAddr: AccountAddress, flags: ASADeploymentFlags
   ): DeployedAssetTxReceipt {
-    return this.addASADef(
+    return this.deployASADef(
       name, this.runtime.loadedAssetsDefs[name], fromAccountAddr, flags
     );
   }
 
   /**
-   * Add Asset without using asa.yaml file
+   * create asset using asa.yaml file
+   * @deprecated `deployASA` should be used instead.
+   * @param name asset name
+   * @param fromAccountAddr account address
+   * @param flags asa deployment flags
+   */
+  addAsset (
```

Comment:
i don't think we need to add this here (in `ctx.ts`). User is not exposed to this function, end user will only use the `runtime` API (eg. `runtime.deployApp`).